<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SigmoidFilterDialog</class>
 <widget class="QDialog" name="SigmoidFilterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>305</width>
    <height>184</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sigmoid filter</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Sigmoid filter parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Output minimum:</string>
        </property>
        <property name="buddy">
         <cstring>spinBoxMinimum</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="spinBoxMinimum">
        <property name="maximum">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QSlider" name="horizontalSliderMinimum">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Output maximum:</string>
        </property>
        <property name="buddy">
         <cstring>spinBoxMaximum</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSpinBox" name="spinBoxMaximum">
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>255</number>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QSlider" name="horizontalSliderMaximum">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximum">
         <number>255</number>
        </property>
        <property name="value">
         <number>255</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Alpha:</string>
        </property>
        <property name="buddy">
         <cstring>spinBoxAlpha</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="spinBoxAlpha">
        <property name="minimum">
         <number>-80</number>
        </property>
        <property name="maximum">
         <number>80</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QSlider" name="horizontalSliderAlpha">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>0</height>
         </size>
        </property>
        <property name="minimum">
         <number>-80</number>
        </property>
        <property name="maximum">
         <number>80</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Beta:</string>
        </property>
        <property name="buddy">
         <cstring>spinBoxBeta</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="spinBoxBeta">
        <property name="minimum">
         <number>-80</number>
        </property>
        <property name="maximum">
         <number>80</number>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QSlider" name="horizontalSliderBeta">
        <property name="minimumSize">
         <size>
          <width>128</width>
          <height>0</height>
         </size>
        </property>
        <property name="minimum">
         <number>-80</number>
        </property>
        <property name="maximum">
         <number>80</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonApply">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonClose">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSliderAlpha</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxAlpha</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>108</y>
    </hint>
    <hint type="destinationlabel">
     <x>129</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderBeta</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxBeta</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>140</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxAlpha</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderAlpha</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>140</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>187</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxBeta</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderBeta</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>134</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>185</x>
     <y>143</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonClose</sender>
   <signal>clicked()</signal>
   <receiver>SigmoidFilterDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>304</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxMinimum</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderMinimum</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>38</y>
    </hint>
    <hint type="destinationlabel">
     <x>179</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxMaximum</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalSliderMaximum</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>137</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderMinimum</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxMinimum</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>39</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>40</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSliderMaximum</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBoxMaximum</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
